editor/debugger/editor_profiler.cpp

```diff
@@ -623,6 +629,13 @@ EditorProfiler::EditorProfiler() {
 	clear_button->set_disabled(true);
 	hb->add_child(clear_button);
 
+	autostart_button = memnew(Button);
+	autostart_button->set_toggle_mode(true);
+	autostart_button->set_text(TTR("Auto Start"));
```

Comment:
Use the same "Autostart" terminology as the Timer node for consistency:

```suggestion
	autostart_button->set_text(TTR("Autostart"));
```